### PR TITLE
Default to building one-job-at-a-time.

### PR DESCRIPTION
**Pull request by ltratt**
One problem with parallel jobs is that we then end up compiling VMs non-deterministically, which may or may not cause unpredictable builds. If "-f" is specified, we do fast builds, otherwise we do one-job-at-a-time "probably safer" builds.

The idea is that, during testing, it's fine to use "-f" but for the real experiment we use the default (slower) setting.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -37,6 +37,36 @@ case `uname` in
...
+
+usage() {
+    echo "Usage: build.sh [-f]" >&2
+    exit 1
+}
+
+# How many cores does this machine have (so we can pass a sensible number to
+# gmake -j)?
+case `uname` in
+    OpenBSD) max_jobs=`sysctl hw.ncpu|cut -d "=" -f 2`;;
```
Comment (vext01) [inline build.sh]: Use `-n` instead of cut?

**Comment by vext01** [comment]
Looks ok.

File: build.sh (hunk 1 above)
Comment (ltratt) [reply build.sh]: I didn't know you could do that. Good idea!

**Comment by ltratt** [comment]
Ready for rereview.

**Comment by vext01** [comment]
Squash and I can merge.

**Comment by ltratt** [comment]
Done.

**Review by vext01 — APPROVED** [review]

**Pull request merged**